dashboards-observability/common/utils/live_tail.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { isEmpty } from 'lodash';
+import datemath from '@elastic/datemath';
+import { DATE_PICKER_FORMAT } from '../../common/constants/explorer';
+import {
+  PPL_INDEX_INSERT_POINT_REGEX,
+  PPL_NEWLINE_REGEX
+} from '../../common/constants/shared';
+
+
+export const insertDateRangeAndSortToQuery = ({
```

Comment:
why not use the existing function? https://github.com/opensearch-project/observability/blob/e24f9823e9f8947e28f04ae01b29eec8b189cfe7/dashboards-observability/common/utils/query_utils.ts#L28

Reply:
existing function didn’t have sort function in it. adding it later was causing an issue so the new function

Reply:
what was the issue? i think at least call the function in yours to reuse the logic, it could cause more issues if we need to always sync implementations in both functions

Reply:
I did try to use the same function by modifying the final query before and even after but nothing seemed to work. the sort function is to be included when final query is composed for live tail, hence the new function

Reply:
@kavithacm Josh removed the parse function form the `preprocessQuery` function. Now you can just add another optional boolean argument `liveTail` and check of if you want to add the sort or not at the end of the final query.